Source/Scene/SkyAtmosphere.js

```diff
@@ -186,17 +233,46 @@ define([
             });
         }
 
+        // Compile the color correcting versions of the shader on demand
+        if (this.colorCorrect() && (!defined(this._spSkyFromSpaceColorCorrect) || !defined(this._spSkyFromAtmosphereColorCorrect))) {
```

Comment:
Call `this.colorCorrect()` just once here in `update` and store it off so it is not computed twice.  Burn a tad less CPU.